src/metabase/query_processor/middleware/add_dimension_projections.clj

```diff
@@ -280,21 +431,23 @@
      ;; Get the entries we're going to add to `:cols` for each of the remapped values we add
      :internal-only-cols (map :new-column internal-only-dims)}))
 
-(defn- add-remapped-cols
+(s/defn ^:private add-remapped-cols
```

Comment:
it annoys me that `add-remapped-cols` and `add-remapped-columns` has basically different semantics and are for basically different parts of the remapping process.

Reply:
that's a good point. I renamed it to make it less confusing